source/databricks/calculation-engine/tests/integration/calculator/calculator_job/conftest.py

```diff
@@ -119,7 +193,8 @@ def executed_wholesale_fixing(
     spark: SparkSession,
     calculator_args_wholesale_fixing: CalculatorArgs,
     migrations_executed: None,
-    test_data_written_to_delta_tables: None,
+    metering_points_and_time_series_input_data_written_to_delta_tables: None,
+    charge_input_data_written_to_delta_tables: None,
```

Comment:
How about something similar to this?
```suggestion
    energy_input_data_written_to_delta: None,
    price_input_data_written_to_delta: None,
```